src/Factory/ClientFactory.php

```diff
@@ -50,43 +51,8 @@ public function create(array $options = [], array $subscribers = [])
         return $client;
     }
 
-    public function createNamed($alias)
```

Comment:
No problem. The only reason why this method is public is because the DI component needed a public callable in order to use it as a factory, and because the factory actually needed to create a registry of listeners.

Regardless, the factory service and methods are deprecated for SF 3.0 (as they are replaced with a DI tag).

As mentioned in `UPGRADE.md` , the factory is no longer the preferred way of creating a client manually anyways.